localstack/services/stepfunctions/asl/antlr/runtime/ASLIntrinsicParser.py

```diff
@@ -117,6 +118,9 @@ def states_func_decl(self):
             return self.getTypedRuleContext(ASLIntrinsicParser.States_func_declContext,0)
 
 
+        def EOF(self):
```

Comment:
That's the main functional change I guess. It's a bit hard to spot within all the re-formatting 🕵️